homeassistant/helpers/translation.py

```diff
@@ -321,3 +330,47 @@ async def async_get_translations(
     for entry in cached:
         result.update(entry)
     return result
+
+
+@bind_hass
+def get_cached_translations(
```

Comment:
We should use our standard naming convention + decorator to show this is OK to call from the event loop:
```suggestion
@callback
def async_get_cached_translations(
```